gitbutler-app/Cargo.toml

```diff
@@ -66,7 +66,7 @@ slug = "0.1.5"
 ssh-key = { version = "0.6.4", features = [ "alloc", "ed25519" ] }
 ssh2 = { version = "0.9.4", features = ["vendored-openssl"] }
 tauri = { version = "1.5.4", features = ["dialog-open", "fs-read-file", "path-all", "process-relaunch", "protocol-asset", "shell-open", "system-tray", "window-maximize", "window-start-dragging", "window-unmaximize"] }
-tauri-plugin-context-menu = { git = "https://github.com/gitbutlerapp/tauri-plugin-context-menu", branch = "main" }
+tauri-plugin-context-menu = { git = "https://github.com/c2r0b/tauri-plugin-context-menu", branch = "main" }
```

Comment:
Well, I kept it as Github URL, because other Dependencies are Git as well. Kind off makes sense as GitButler is in Beta Phase, it is always nice to have latest version straight from the repository.

Once GitButler stabilise itself, it would be good to use stable version. 

But obviously, management has to decide on this.